src/App.js

```diff
@@ -10,7 +18,14 @@ class App extends React.Component {
 
   render() {
     return (
-      <Calculator />
+      <Router>
+        <Nav />
+        <Routes>
+          <Route path="/" element={<Home />} />
```

Comment:
- [Optional ] Kindly I recommend you to make  the Home page default when the page start , it show empty page So Kindly fix it Just like this :  `Route path="/math-magicians" element={<Home />}` So when you modify it here so change path on the Nav-bar just like the above for Home page.

![Capture](https://user-images.githubusercontent.com/93598189/213144080-920405d8-ad2e-4437-9a93-9477abb98682.PNG)